qiskit/pulse/channels/output_channel.py

```diff
@@ -55,41 +105,50 @@ class DriveChannel(OutputChannel):
 
     prefix = 'd'
 
-    def __init__(self, index: int, lo_frequency: float = None):
+    def __init__(self, index: int,
+                 lo_frequency: float = None,
+                 lo_freq_range: Tuple[float, float] = (0, float("inf"))):
```

Comment:
Can this just be `None`? We can set `self._lo_freq_range = None`, or maybe `NoLoRestriction()` for typecheck?

Reply:
This is like returning [] instead of None. I think it's better to return infinite range.